project_mis_report_budget/models/mis_budget.py

```diff
@@ -9,3 +9,8 @@ class MisBudget(models.Model):
 
     project_id = fields.Many2one('project.project')
     analytic_account_project_id = fields.Many2one('account.analytic.account', related='project_id.analytic_account_id')
+    is_margin = fields.Boolean()
```

Comment:
This field should be readonly, as it shouldn't be filled by users (is a technical field). And, is it possible to be added at least to one view (e.g. custom tree view within projects) and set it visible only in debug mode, for testing purposes?